composer.json

```diff
@@ -31,23 +30,26 @@
         "robojackets/apiary-user": "1.0.3",
         "robojackets/auth-stickler": "0.3",
         "saumini/count": "1.0.1",
+        "sentry/sentry-laravel": "^2.4",
         "spatie/laravel-permission": "3.18.0",
         "spatie/laravel-query-builder": "3.3.4",
         "spatie/once": "2.2.1",
         "staudenmeir/eloquent-has-many-deep": "*",
         "subfission/cas": "4.0.0",
+        "symfony/polyfill-uuid": "^1.22",
```

Comment:
I don't think this package is actually being installed, because it is in the `replace` configuration.

Is the real `uuid` extension available in the RHEL/Ubuntu repos?

Reply:
You'd have to install PECL via `php-pear`, then `pecl install uuid`